### DISCUSSION TITLE
try to create multiple window in tauri 2 but get 400

### DISCUSSION THREAD

**Discussion by DexYang** [tauri-apps/tauri#9487] [Q&A] [answered]
i copy this code from [Guides](https://beta.tauri.app/references/v1/js/namespacewindow/#webviewwindow) in App.vue

```
import { WebviewWindow } from "@tauri-apps/api/webviewWindow"

const webview = new WebviewWindow('my-label', {
  url: 'https://github.com/tauri-apps/tauri',
  width: 200,
  height: 800
});
webview.once('tauri://created', function () {
  // webview window successfully created
});
webview.once('tauri://error', function (e) {
  // an error happened creating the webview window
});
```

get this error in console

```
Failed to load resource: the server responded with a status of 400 (Bad Request)  "http://ipc.localhost/plugin%3Awebview%7Ccreate_webview_window"
```

what am i doing wrong....

**Comment by FabianLars** [comment, marked as answer]
you should have a .json file in src-tauri/capabilities folder. Open that and there should be a `permissions` array. Just add `"webview:allow-create-webview-window"` to that and it should work.

Please bear with us while we're working on the docs...

**Reply by nomandhoni-cs** [reply]
Thanks Man This worked for me

**Reply by Rafferty97** [reply]
It looks like this has been changed to "core:webview:allow-create-webview-window"? Just a note for anyone else stumbling upon this thread.

**Reply by GheorgheP** [reply]
But the error still points to the old permission name. @Rafferty97, thank you for pointing out, you saved me from other few hours pain.

**Reply by victormongi** [reply]
> It looks like this has been changed to "core:webview:allow-create-webview-window"? Just a note for anyone else stumbling upon this thread.

hello where can I get this permissions list?
thank you

**Reply by justingolden21** [reply]
@victormongi src-tauri/capabilities/migrated.json, under "permissions" which is an array

![image](https://github.com/user-attachments/assets/baba708c-c85b-4df2-bcd7-ab1b4494283b)

**Reply by justingolden21** [reply]
The error message should definitely be fixed. Without finding this specific thread online, I would have never known. I followed the error's advice and got more errors and got stuck.

**Comment by DexYang** [comment]
That works, thx!

**Comment by inksnowhailong** [comment]
It didn't work again. I added relevant content, but it still reported an error: "webview.create_webview not allowed. Permissions associated with this command: webview:allow-create-webview"
Added a toml file according to the official website and tried again, but it didn't work either.

**Reply by ogrnz** [reply]
I added `"core:webview:allow-create-webview-window"` in the permissions  of my `default.json` capabilities file. That did the trick for me.

**Comment by Gaimo** [comment]
What am I doing wrong? Windows 11, Tauri V2

error:
```
event: "tauri://error"
id: -1
payload: "webview.create_webview not allowed. Permissions associated with this command: webview:allow-create-webview"
```

default.json
```json
{
  "$schema": "../gen/schemas/desktop-schema.json",
  "identifier": "default",
  "description": "Capability for the main window",
  "windows": ["main"],
  "permissions": [
    "core:default",
    "opener:default",
    "core:window:allow-start-dragging",
    "core:webview:allow-create-webview-window"
  ]
}
```

**Reply by nomandhoni-cs** [reply]
that means you have to add this permission also `webview:allow-create-webview`
